### PR TITLE
Cleanup bigint literals

### PR DESCRIPTION
**Pull request by erights**

### CODE REVIEW

File: packages/zoe/src/contractSupport/bondingCurves.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { natSafeMath } from './safeMath';
 
 const { multiply, floorDivide } = natSafeMath;
 
-// TODO(2196): use 10000n instead of  BigInt(10000)
+// TODO(2196): use 10000n instead of  10000n
```
Comment (Chris-Hibbert) [inline packages/zoe/src/contractSupport/bondingCurves.js]: The issue is already closed, so you can just drop this comment.

**Review by Chris-Hibbert — APPROVED** [review]
Looks good. One TODO (to update to BigInts) should be dropped.

File: packages/transform-metering/src/meter.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ const { ceil } = Math;
 const ObjectConstructor = Object;
 
 // eslint-disable-next-line no-bitwise
```
Comment (Chris-Hibbert) [inline packages/transform-metering/src/meter.js:8]: This eslint-disable comment isn't needed.  Nor is the one on line 1.

Comment (erights) [reply packages/transform-metering/src/meter.js:8]: Oops, missed this. Addressed in #2309

**Pull request merged**